From 52f4da1d90ef6a6c56d7e6016539c0054066c458 Mon Sep 17 00:00:00 2001
Message-Id: <52f4da1d90ef6a6c56d7e6016539c0054066c458.1387276076.git.minovotn@redhat.com>
In-Reply-To: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
References: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 10 Dec 2013 15:12:52 +0100
Subject: [PATCH 11/16] milkymist-minimac2: Fix minimac2_read/_write
 tracepoints

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1386688376-29521-4-git-send-email-armbru@redhat.com>
Patchwork-id: 56111
O-Subject: [PATCH 7.0 qemu-kvm 3/7] milkymist-minimac2: Fix minimac2_read/_write tracepoints
Bugzilla: 997832
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

Broken in milkymist-minimac.c from the start (commit 0742454),
faithfully moved to milkymist-minimac2.c (commit 57aa265).

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 0ece9671fd22e3ea518aa30f941e8794116985e7)
---
 hw/net/milkymist-minimac2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/net/milkymist-minimac2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/net/milkymist-minimac2.c b/hw/net/milkymist-minimac2.c
index 4ef6318..7448777 100644
--- a/hw/net/milkymist-minimac2.c
+++ b/hw/net/milkymist-minimac2.c
@@ -355,7 +355,7 @@ minimac2_write(void *opaque, hwaddr addr, uint64_t value,
 {
     MilkymistMinimac2State *s = opaque;
 
-    trace_milkymist_minimac2_memory_read(addr, value);
+    trace_milkymist_minimac2_memory_write(addr, value);
 
     addr >>= 2;
     switch (addr) {
-- 
1.7.11.7

